src/components/shared/Dialog/dialogTypes/CropDialog.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+import ReactCrop from 'react-image-crop';
+import 'react-image-crop/dist/ReactCrop.css';
+
+class CropDialog extends Component {
+  static propTypes = {
+    title: PropTypes.string,
+    content: PropTypes.string,
+    handleConfirm: PropTypes.func,
+    imagePreviewUrl: PropTypes.string,
+    handleClose: PropTypes.func,
+  };
+
+  state = {
+    src: this.props.imagePreviewUrl,
+    crop: {
+      unit: '%',
+      width: 30,
+      aspect: 1 / 1,
+    },
+    croppedImageUrl: null,
+    imageBase64: null,
+  };
+
+  getCroppedImg = (image, crop, fileName) => {
+    const canvas = document.createElement('canvas');
+    const scaleX = image.naturalWidth / image.width;
+    const scaleY = image.naturalHeight / image.height;
+    canvas.width = crop.width;
+    canvas.height = crop.height;
+    const ctx = canvas.getContext('2d');
+
+    ctx.drawImage(
+      image,
+      crop.x * scaleX,
+      crop.y * scaleY,
+      crop.width * scaleX,
+      crop.height * scaleY,
+      0,
+      0,
+      crop.width,
+      crop.height,
+    );
+
+    let imageBase64 = canvas.toDataURL('image');
```

Comment:
`imageBase64`  contains the image data in `base64` format which is passed to `AccountTab.react.js` component using function `handleConfirm`.